inspirehep/alembic/c6570e49b7b2_change_record_citations_to_records_.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Change record_citations to records_citations for consistency"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "c6570e49b7b2"
+down_revision = "5ce9ef759ace"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+    op.rename_table("record_citations", "records_citations")
+
+
+def downgrade():
+    """Downgrade database."""
+    op.rename_table("records_citations", "record_citations")
```

Comment:
also the name of the indexes should be changed to follow the sqlalchemy naming convention:
* `idx_citations_cited` -> `ix_records_citations_cited_id`
* `fk_record_citations_cited` -> `fk_records_citations_cited_id`
* `fk_record_citations_citer` -> `fk_records_citations_citer_id`